PeerioTechnologies/peerio-desktop
# fix: NewContact search context

#### Relevant info and issue/PR links 
 https://app.clubhouse.io/peerio/story/8902/mc-unable-to-add-peerio-users-on-add-user-page

#### Testing instructions  
In MedCryptor, NewContact view should allow addition of both Peerio and MedCryptor users
* Note that non-chat snackbars are broken on devl still, fixed in this merge https://github.com/PeerioTechnologies/peerio-desktop/pull/322
* to test, add the user and then immediately navigate to an active chat and the snackbar there should show the "user added" message

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
